src/events/keyboard.js

```diff
@@ -195,6 +194,117 @@ function keyboard(p5, fn){
    */
   fn.key = '';
 
+  /**
+   * The `code` property represents a physical key on the keyboard (as opposed 
+   * to the character generated by pressing the key). In other words, this 
+   * property returns a value that isn't altered by keyboard layout or the state 
+   * of the modifier keys.
+   * 
+   * This property is useful when you want to handle keys based on their 
+   * physical positions on the input device rather than the characters associated 
+   * with those keys;
+   *
+   * Unlike <a href="#/p5/key">key</a>, the `code` property differentiates between 
+   * physical keys that generate the same character—for example, `CtrlLeft` and 
+   * `CtrlRight`—so each can be handled independently.
+   * 
+   * Pressing the key physically labeled “A” always yields `KeyA`, regardless 
+   * of the current keyboard layout (QWERTY, Dvorak, AZERTY, etc.) or the character 
+   * that appears in a text field.
+   * 
+   * The code property returns a plain string (e.g., 'ArrowRight'). You can 
+   * compare it directly with string literals:
+   * ```js
+   *  if (code === 'ArrowRight') {
+   *  // …
+   * }
+   * ```
+   * 
+   * For extra clarity, you can instead use the predefined key-code constants 
+   * provided by p5.js—`BACKSPACE`, `DELETE`, `ENTER`, `RETURN`, `TAB`, `ESCAPE`, `SHIFT`, 
+   * `CONTROL`, `OPTION`, `ALT`, `UP_ARROW`, `DOWN_ARROW`, `LEFT_ARROW`, and `RIGHT_ARROW`. 
+   * These are simply shorthands for the same string values:
+   * ```js
+   * if (code === RIGHT_ARROW) {
+   * // ..
+   * }
+   * ```
+   * 
+   * @property {String} code
+   * @readOnly
+   *
+   * @example
+   * <div>
+   * <code>
+   * // Click on the canvas to begin detecting key presses.
+   *
+   * function setup() {
+   *   createCanvas(100, 100);
+   *
+   *   describe(
+   *     'A gray square. The last key pressed is displayed at the center.'
+   *   );
+   * }
+   *
+   * function draw() {
+   *   background(200);
+   *
+   *   // Style the text.
+   *   textAlign(CENTER);
+   *   textSize(16);
+   *
+   *   // Display the last key pressed.
+   *   text(code, 50, 50);
+   * }
+   * </code>
+   * </div>
+   *
+   * <div>
+   * <code>
+   * // Click on the canvas to begin detecting key presses.
+   *
+   * let x = 50;
+   * let y = 50;
+   *
+   * function setup() {
+   *   createCanvas(100, 100);
+   *
+   *   background(200);
+   *
+   *   describe(
+   *     'A gray square with a black circle at its center. The circle moves when the user presses an arrow key. It leaves a trail as it moves.'
+   *   );
+   * }
+   *
+   * function draw() {
+   *   // Update x and y if an arrow key is pressed.
```

Comment:
What do you think about using `keyIsDown` here as well? https://editor.p5js.org/ksen0/sketches/vrMANhunx Main benefit: backwards compatible snippet. Main drawback: makes this reference page less standalone / use another function. Up to you if you think it's a good idea!